tcf_website/static/base/base.css

```diff
@@ -97,3 +83,20 @@ footer > .row {
 .form-row {
   margin: 0.5rem;
 }
+
+footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 80px; /* Set the fixed height of the footer here */
+    line-height: 30px; /* Vertically center the text there */
+    /* background-color: #f5f5f5; */
```

Comment:
An unnecessary comment?

Reply:
I'll delete the commented out background color line

I didn't make the rest of those comments (just rearranged the code), but I think it's fine?

Reply:
Yeah, I was referring only to `background-color`.